windows/window.c

```diff
@@ -2727,6 +2727,8 @@ static LRESULT CALLBACK WndProc(HWND hwnd, UINT message,
          */
         noise_ultralight(NOISE_SOURCE_MOUSEPOS, lParam);
 
+        // TODO: Handle the mouse move when "Any-event tracking" (1003 DECSET)
```

Comment:
Is this comment is up-to-date or actually done below? In the later case move it there and drop the TODO.